src/index.ts

```diff
@@ -882,11 +884,41 @@ export const item = {
 			url: string;
 			vault: string;
 		}> = {},
-	) =>
-		cli.execute<Item>(["item", "create"], {
+	) => {
+		const options: {
+			flags: Flags;
+			args?: FieldAssignment[];
+			stdin?: Record<string, any>;
+		} = {
 			flags,
-			args: assignments,
-		}),
+		};
+
+		// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-call
+		const version = semverCoerce(cli.getVersion());
+
+		if (semverSatisfies(version, ">=2.6.2")) {
```

Comment:
I'd recommend adding a comment explaining why we're branching behavior based on CLI version.

Reply:
Oh yes great callout!